code_search/kubeflow/components/export-model.libsonnet

```diff
@@ -0,0 +1,68 @@
+{
+  parts(params, env):: {
+    job: {
+      apiVersion: "batch/v1",
+      kind: "Job",
+      metadata: {
+        name: params.name,
+        namespace: env.namespace,
+        labels: {
+          app: params.name,
+        },
+      },
+      spec: {
+        replicas: 1,
+        template: {
+          metadata: {
+            labels: {
+              app: params.name,
+            },
+          },
+          spec: {
+            // Don't restart because all the job should do is launch the Dataflow job.
+            restartPolicy: "Never",
+            containers: [
+              {
+                name: "exporter",
+                image: params.image,
+                command: [
+                  "t2t-exporter",
+                  "--problem=" + params.problem,
+                  "--data_dir=" + params.dataDir,
+                  // TODO(jlewi): If we specify checkpoint path then we can specify a particular
+                  // checkpoint. Does this then allow us to specify a different directory for outputDir?
```

Comment:
I think with t2t-exporter you can specify the export dir via --export or --export_dir instead of it going to the default FLAGS.output_dir/export, if that's what you were getting at.

Reply:
Thanks @cwbeitel 
Filed /kubeflow/examples#331

I'll fix it in a follow on PR because I want to get this PR submitted to unblock @IronPan